docs/user-guide/config-examples.md

```diff
@@ -0,0 +1,30 @@
+# User configurations
+
+Want some help with starting your configuration ?
+Have a look at this catalog of configuration examples.
+
+<!-- TODO: add info about `kickstart.nixvim` -->
+
+**Note:**\
+Most of those configurations are using a [standalone flake](../modules/standalone.html).\
```

Comment:
Might be misleading to conflate standalone builds with seperate flakes? E.g. I use a standalone build, but it's just one flake output on my main system configs flake. While maybe not in these examples, someone _could_ do a standalone build without using flakes at all... Or maybe I'm just being pedantic?

```suggestion
Most of those configurations are using a [standalone build](../modules/standalone.html).\
```

Reply:
I'd probably also replace the period with a comma and the next line's "of course" with a "however" to join the two sentences into a single thought.

I'm getting very picky and subjective here though.

Reply:
> I'd probably also replace the period with a comma and the next line's "of course" with a "however" to join the two sentences into a single thought.

Done



> I'm getting very picky and subjective here though.

No, this is useful I think. As a native English speaker, you definitely have your say in this !

Reply:
> Might be misleading to conflate standalone builds with seperate flakes? E.g. I use a standalone build, but it's just one flake output on my main system configs flake. While maybe not in these examples, someone _could_ do a standalone build without using flakes at all... Or maybe I'm just being pedantic?

While I completely get your point, I would stay with "standalone flake" to ensure consistency with the rest of our documentation (website, README, issues, ...). I agree that the term is not technically accurate.
Any opinion @traxys ?

Reply:
I think I agree with @MattSturgeon, we should try to dissociate flake & build

Reply:
Sure: `flake` -> `build` done

Reply:
Is that fine for you like this @MattSturgeon ?

Reply:
> Is that fine for you like this?

Yep, that wording lgtm